software/base/src/main/java/brooklyn/entity/brooklynnode/BrooklynNodeSshDriver.java

```diff
@@ -127,8 +127,8 @@ public void install() {
         String uploadUrl = entity.getConfig(BrooklynNode.DISTRO_UPLOAD_URL);
         
         // Need to explicitly give file, because for snapshot URLs you don't get a clean filename from the URL.
-        // This filename is used to generate the first URL to try: 
-        // file://$HOME/.brooklyn/repository/BrooklynNode/0.6.0-SNAPSHOT/brooklyn-0.6.0-SNAPSHOT-dist.tar.gz
+        // This filename is used to generate the first URL to try: [BROOKLYN_VERSION_BELOW]
+        // file://$HOME/.brooklyn/repository/BrooklynNode/0.8.0-SNAPSHOT/brooklynnode-0.8.0-snapshot.tar.gz
```

Comment:
Yes, the filename is lowercase (I copied the URL from stdin). See https://github.com/apache/incubator-brooklyn/blob/master/core/src/main/java/brooklyn/entity/drivers/downloads/DownloadProducerFromLocalRepo.java#L51.

Good point about `BROOKLYN_VERSION_BELOW`, the lower case part won't get replaced, so I think best is to just remove the marker and leave the version as is.